test/pacignore/rm_success.t

```diff
@@ -64,7 +64,7 @@ Check successful rm with options in between and additional IgnorePkg
 Check successful rm with options in between and additional IgnorePkg
 
   $ PACMAN_CONF_TEST="$(write_pacman_conf "[options]" "# SomeOption" "IgnorePkg = foo" "# AnotherOption" "IgnorePkg = bar")"
-  > fakeroot pacignore "rm" "--pacman-conf" "$PACMAN_CONF_TEST" "foo" "bar"
+  > fakeroot pacignore "rm" "-c" "$PACMAN_CONF_TEST" "foo" "bar"
```

Comment:
This was existing, and no need to muddy the PR by changing it, but it's a little odd to see sub-commands / options quoted like this. `pacignore rm -c "$PACMAN_CONF_TEST" foo bar` would be more idiomatic.

Reply:
I'll get back to this in a separate PR